apps/lockfile-explorer-server/src/exampleData/package.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Instead of checking this second copy of this file in, can you just copy it from `node_modules/@rushstack/heft/package.json` to `lib/exampleData` using a `heft.json` eventAction?

Reply:
This file is currently only being used as testing data, and will be removed in a later update